target/linux/zynq/config-5.4

```diff
@@ -369,7 +369,6 @@ CONFIG_JBD2=y
 # CONFIG_JFFS2_COMPRESSION_OPTIONS is not set
 CONFIG_JFFS2_ZLIB=y
 CONFIG_KERNEL_GZIP=y
-# CONFIG_KERNEL_MODE_NEON is not set
```

Comment:
How come you don't need an explicit =y line here? Other parts of the build system add it?

Reply:
Yeah, it inherits from `target/linux/generic/config-5.{4,10}`